userguide/analytics/userguide_analytics.adoc

```diff
@@ -1,63 +1,595 @@
+----
+
+== Data API
+
+The analytics plugin provides a Data API that supports some account level operations as explained below.
+
+=== Retrieve Analytics Data for Account
+
+You can retrieve all analytics data for an account as follows:
+
+[source,bash]
+----
+curl -u admin:password \
+     -H "X-Killbill-ApiKey:bob" \
+     -H "X-Killbill-ApiSecret:lazar" \
+     "http://127.0.0.1:8080/plugins/killbill-analytics/<account_id>"
+----
+
+This returns the data from the analytics tables for the specified `account_id`.
+
+=== Refresh an Account
+
+You can force a refresh for a given account as follows:
+
+[source,bash]
+----
+curl -v \
+     -X PUT \
+     -u admin:password \
+     -H "X-Killbill-ApiKey:bob" \
+     -H "X-Killbill-ApiSecret:lazar" \
+     "http://127.0.0.1:8080/plugins/killbill-analytics/<account_id>"
+----
+
+This refreshes the analytics tables for the specified `account id`.
+
+=== Refresh all Accounts
+
+You can refresh all accounts as follows:
+
+[source,bash]
+----
+curl -s \
+     -u admin:password \
+     -H "X-Killbill-ApiKey:bob" \
+     -H "X-Killbill-ApiSecret:lazar" \
+     "http://127.0.0.1:8080/1.0/kb/accounts/pagination" | \
+ruby -r json -e 'JSON.parse(gets).map { |a| puts a["accountId"] }' | \
+xargs -I accountId \
+curl -v \
+     -X PUT \
+     -u admin:password \
+     -H "X-Killbill-ApiKey:bob" \
+     -H "X-Killbill-ApiSecret:lazar" \
+    "http://127.0.0.1:8080/plugins/killbill-analytics/accountId"
+----
+
+[[analytics_tables]]
+== Analytics Tables
+
+This section gives a brief overview of some of the important tables in the analytics database.
+
+=== Accounts
+
+The table *analytics_accounts* maintains information about Kill Bill accounts. The table doesn't contain historic records (it will contain at most one record per Kill Bill account).
+
+=== Subscriptions
+
+The table *analytics_subscription_transitions* maintains information about Kill Bill subscriptions. It describes the history of changes (both past and future) on a per subscription basis. It contains all phase transitions for all subscriptions and can be used to build subscription timelines. A few remarks:
+
+* Given a subscription id, events are sorted chronologically when stored (you can rely on `record_id asc`)
+* *prev_* fields will always be `NULL` for creation events
+* *next_* fields will always be `NULL` for cancellation events
+* A `CHANGE` event corresponds to a user initiated change (e.g. upgrade) whereas `SYSTEM_CHANGE` corresponds to a phase event (e.g. transition from trial to evergreen phase)
+* The *requested_timestamp* field reflects when the transition was requested, the *next_start_date* field reflects the effective transition time
+
+=== Invoices
+
+The tables *analytics_invoices*, *analytics_invoice_adjustments*, *analytics_invoice_item_adjustments*, *analytics_invoice_credits* and *analytics_invoice_items* maintain information about invoices posted by account.
+
+A few remarks:
+
```

Comment:
I'd put a period at the end of each bullet item.

Reply:
Fixed